src/app/_components/Badge.tsx

```diff
@@ -1,12 +1,24 @@
 type BadgeProps = {
-  featured: boolean
-  children?: string
+  children: React.ReactNode
+  badgeType: 'primary' | 'ghost'
 }
 
-export function Badge({ featured, children }: BadgeProps) {
-  return (
-    <span className="inline-block px-2 py-1 bg-white text-brand-800 uppercase rounded-xl text-sm font-semibold">
-      {featured ? 'Featured Project' : children}
-    </span>
-  )
+export function Badge({ children, badgeType }: BadgeProps) {
```

Comment:
Same comment for `children`

Reply:
Sorry what do you mean by this?

Reply:
Passing `childeren` last:

`export function Badge({ prop1, prop2, children }: BadgeProps) {`